tests/btrfs_test.py

```diff
@@ -36,6 +35,9 @@ def setUpClass(cls):
         else:
             BlockDev.reinit(cls.requested_plugins, True, None)
 
+
+class BtrfsTestCase(BtrfsTest):
```

Comment:
I think these deserve a better name. If someone goes and adds a new test here, they can easily use `BtrfsTest` as the base class.

Reply:
I plan to do a big test "cleanup" before the 3.0 release, there is a lot of outdated/messy things there. I kept this simple, we have `XXXTestCase` and `XXXTest` for other plugins too.